context_builder.Dockerfile

```diff
@@ -10,30 +10,28 @@ RUN apt-get update \
     && apt-get clean -y \
     && rm -rf /var/lib/apt/lists/* /tmp/library-scripts/
 
-RUN curl -sSL https://install.python-poetry.org | python -
-ENV PATH="/root/.local/bin:${PATH}"
+RUN curl -LsSf --retry 5 --retry-delay 5 https://astral.sh/uv/0.7.12/install.sh | sh
```

Comment:
We don't really use the context builder anymore, but it's easier to just make the test/build work here rather than ripping it all out in this migration PR. A follow-up should continue to deprecate our building/publishing of these deprecated images